### ISSUE TITLE
Can I combine leading @ with interpolated strings

### ISSUE THREAD

**Issue by nojaf** [dotnet/docs#23806]
Hello,

Is the following possible in F#?
```fsharp
printfn $@"Migrate notes of file ""{oldId}"" to new file ""{newId}""."
```

The combination of `$` and `@`.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 178dbf92-8d00-887f-f55c-44ae4ce0fc26
* Version Independent ID: fa136057-4782-c009-437b-f08bdd267f6a
* Content: [Interpolated strings - F#](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/interpolated-strings)
* Content Source: [docs/fsharp/language-reference/interpolated-strings.md](https://github.com/dotnet/docs/blob/main/docs/fsharp/language-reference/interpolated-strings.md)
* Product: **dotnet-fsharp**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**

---
[Associated WorkItem - 494891](https://dev.azure.com/msft-skilling/Content/_workitems/edit/494891)

**Comment by cartermp** [comment]
Yep, this should be added. It's got the same rules as `@`-encoded strings that aren't interpolated, where the `\` character is ignored.
